tools/torch_quant/torch_quant/quantizer.py

```diff
@@ -175,6 +176,8 @@ def quantize_gm(self, gm: GraphModule, root: nn.Module) -> None:
             ])
         else:
             raise ValueError(f'Unsupported backend {self.backend.name}')
+        # remove unused modules (e.g. observers) or the following tracing might fail
+        ctx.gm.delete_all_unused_submodules()
```

Comment:
When will redundant modules appear?

Reply:
If backend is fbgemm or reference, we didn't remove the observers after converting them to quantize/dequantize ops.